src/controllers/authController.ts

```diff
@@ -87,23 +109,16 @@ class AuthController {
 		//Hash the password, to securely store on DB
 		user.hashPassword();
 
-		//Try to save. If fails, the username is already in use
+		//Try to save. If fails, the email is already in use
 		try {
 			await userService.save(user);
 		} catch (e) {
-			return res.status(409).send('username already in use');
+			return res.status(409).send('email already in use');
```

Comment:
```suggestion
			return res.status(409).send(e);
```

måske ændre e til error som resten af koden